Resources/Prototypes/Entities/Objects/Weapons/Guns/flare_gun.yml

```diff
@@ -1,39 +1,96 @@
 - type: entity
   name: flare gun
-  parent: BaseWeaponLauncher
+  parent: BaseItem
   id: WeaponFlareGun
   description: A compact, single-shot pistol that fires shotgun shells.
   components:
   - type: Sprite
     sprite: Objects/Weapons/Guns/Shotguns/flaregun.rsi
     layers:
-      - state: icon
+      - state: base
```

Comment:
Avoid renaming textures, it can break things downstream

Reply:
This is required for the system that changes the sprite depending if the "bolt" is open or closed.

Reply:
I can make it so theres both a base AND an icon texture, but we need one called base for said system.

Reply:
OK, I'll look into what should be done here

Reply:
[It looks like it should be fine to rename](https://discord.com/channels/310555209753690112/900426319433728030/1301401658072367115)